napari/_qt/menus/file_menu.py

```diff
@@ -123,7 +123,7 @@ def __init__(self, window: 'Window') -> None:
                 'shortcut': 'Ctrl+W',
             },
             {
-                'when': running_as_bundled_app(),
+                'when': running_as_constructor_app(),
```

Comment:
Is the Restart button a UX feature or something we needed in Briefcase for XYZ reasons?